site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -65,18 +65,41 @@ const ReviewDisruption = ({ disruption, previousSocialMediaPosts, csrfToken }: R
                 disruptionStartTime: disruption.disruptionStartTime,
                 disruptionEndDate: disruption.disruptionEndDate,
                 disruptionEndTime: disruption.disruptionEndTime,
+                disruptionRepeats: disruption.disruptionRepeats,
+                disruptionRepeatsEndDate: disruption.disruptionRepeatsEndDate,
             },
         ];
 
-        return validity.map((validity, i) => ({
-            header: `Validity period ${i + 1}`,
-            cells: [
-                validity.disruptionEndDate && validity.disruptionEndTime && !validity.disruptionNoEndDateTime
-                    ? `${validity.disruptionStartDate} ${validity.disruptionStartTime} - ${validity.disruptionEndDate} ${validity.disruptionEndTime}`
-                    : `${validity.disruptionStartDate} ${validity.disruptionStartTime} - No end date/time`,
-                createChangeLink(`validity-period-${i + 1}`, "/create-disruption"),
-            ],
-        }));
+        return validity.map((validity, i) => {
+            const appendValue =
+                validity.disruptionRepeats === "daily" ? (
+                    <>
+                        <br />
+                        Repeats {validity.disruptionRepeats} until {validity.disruptionRepeatsEndDate}
+                    </>
+                ) : validity.disruptionRepeats === "weekly" ? (
+                    <>
+                        <br />
+                        Repeats every week until {validity.disruptionRepeatsEndDate}
+                    </>
+                ) : (
+                    <></>
+                );
```

Comment:
What are the <br> 's for?

Reply:
<br/> is for line break. The text following br should be displayed in next line

Reply:
If we just had 
```suggestion
            Repeats {validity.disruptionRepeats} until {validity.disruptionRepeatsEndDate}
```

then it would work for both daily and weekly, and we wouldnt need 2 versions.

Reply:
And ideally we dont need to use <br /> - the append value just needs to be a string, and we can do the html stuff in the next part. Then we arent adding html to html.

Reply:
Was told that the text should be of below format. Above suggestion will not display "every week"

"Repeats {"daily"/"every week"} until {Ending on}"